.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,23 @@
+#### Description:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)